ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.cs

```diff
@@ -238,6 +238,9 @@ public override void ResetDocument()
 
     #region receiving 
     public override bool CanPreviewReceive => true;
+
+    private static bool IsPreviewIgnore(Base @object) => @object.speckle_type.Contains("Block") || @object.speckle_type.Contains("View");
```

Comment:
additionally, I noticed that this check was being performed in 3 places. so extracted a method.